src/ontology/edits/oeo-shared.omn

```diff
@@ -2535,6 +2652,23 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1253",
         OEO_00020154
     
     
+Class: OEO_00010296
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A primary energy consumption calculation method is a methodology to calculate primary energy consumption.",
```

Comment:
Hm, not the smartest definition... 🙄

Reply:
If just moved the class, but did not change it. If you want to improve this definition, please open an issue for that.